dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/datafile/ingest/IngestError.java

```diff
@@ -39,18 +39,14 @@
 
     // -------------------- LOGIC --------------------
 
-    public String getErrorMessage() {
-        return BundleUtil.getStringFromBundle(ERROR_KEY_PREFIX + toString());
-    }
-
     public String getErrorMessage(List<String> arguments) {
-        return arguments.isEmpty() ? getDefaultErrorMessage() : BundleUtil.getStringFromBundle(ERROR_KEY_PREFIX + toString(), arguments.toArray());
+        return arguments.isEmpty() ? getErrorMessage() : BundleUtil.getStringFromBundle(ERROR_KEY_PREFIX + toString(), arguments.toArray());
```

Comment:
I don't think that there is any difference between:
`BundleUtil.getStringFromBundle(ERROR_KEY_PREFIX + toString(), arguments.toArray())` == `BundleUtil.getStringFromBundle(ERROR_KEY_PREFIX + toString())`

Reply:
Oh you mean if it's empty? I need to check it then.